appservice/src/main/resources/static/swagger/swagger-spec.yaml

```diff
@@ -2106,6 +2200,15 @@ definitions:
         - DisclosureResponseMessage
         - ActivityCommentMessage
         - SystemMessage
+      _links:
+         allOf:
+           - $ref: '#/definitions/Curies'
+           - $ref: '#/definitions/SelfLink'
+           - $ref: '#/definitions/UserPhotoLink'
+      nickname:
+        type: string
+        description: The nickname of the buddy, or a self or system sender placeholder
```

Comment:
In case of a system message, there won't be a user photo link, I guess. Shouldn't we add a ticket to return a fixed image in that case? Could simply be the Yona logo.

Reply:
Shouldn't the app itself take care of that?

Reply:
Given that all photo's come from the server, it makes sense to just provide a uniform contract to them.

They know the concept of a system message, so it's indeed also possible to use a static image, part of the app. Let's leave it to them at least for now. If they ask for it, it would be simple to add it.